src/Integration.Vsix/SonarLintDaemonPackage.cs

```diff
@@ -135,8 +136,7 @@ protected override void Dispose(bool disposing)
 
             if (disposing)
             {
-                vcxCompilationDatabaseStorage?.Dispose();
-                vcxCompilationDatabaseStorage = null;
+                DisposeCompilationDatabaseStorageAsync().Forget();
```

Comment:
This means that there is a chance we won't delete the file, because the process might shut down before this is complete

Reply:
Call it synchronously, or remove the DropDatabase call completely (see https://github.com/SonarSource/sonarlint-visualstudio/pull/6244#discussion_r2126144625)